packages/main/test-integration/web/index.test.ts

```diff
@@ -150,11 +168,7 @@ describe("startChat", function () {
     const question1 = "What is the capital of Oregon?";
     const question2 = "How many people live there?";
     const question3 = "What is the closest river?";
-    const chat = model.startChat({
-      generationConfig: {
-        maxOutputTokens: 100,
-      },
```

Comment:
curious, is there a reason to remove the max tokens limit here?

Reply:
Yeah, so this is a live backend test so it doesn't always come back with the same results so it's very loose in what it considers a success (any non-empty string). Lately, it will come back with nothing (empty string) if I set the token limit to 100, and I'm not sure any number guarantees something will come back, so I just removed it.